react/features/app/components/AbstractApp.js

```diff
@@ -253,10 +252,10 @@ export class AbstractApp extends BaseApp<Props, *> {
             }));
         this.nativeEventListeners.push(videoConfBridgeEmitter.addListener(NativeEvents.ADD_COMMAND_LISTENER,
             (commandName: string) => dispatch(addCommandListener(commandName))));
-
-        // TODO replace with appropriate logic. Received value can be a boolean, or number 0 or 1.
+        this.nativeEventListeners.push(videoConfBridgeEmitter.addListener(NativeEvents.ADD_COMMAND_LISTENER,
```

Comment:
I think a rebase conflict solving went wrong, this listener is duplicated with the rows above